server/src/ai/nl-graph-query/nl-graph-query.service.ts

```
@@ -106,6 +106,11 @@ export class NlGraphQueryService {
       const explanation = request.verbose
         ? explainQuery(cypher, true)
         : summarizeQuery(cypher);
+      const explanationDetails = buildQueryExplanation(cypher, {
+        warnings,
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)